_data/meltano/extractors/tap-theme-parks/danielpdwalker.yml

```
@@ -16,6 +16,7 @@ name: tap-theme-parks
 namespace: tap_theme_parks
 pip_url: git+https://github.com/DanielPDWalker/tap-theme-parks.git
 repo: https://github.com/DanielPDWalker/tap-theme-parks
+quality: bronze
```

```suggestion
quality: bronze
repo: https://github.com/DanielPDWalker/tap-theme-parks
```

Oops sorry my first suggestion wasnt in alpha order 😢